01-messaging-crypto-and-init.md

```diff
@@ -20,24 +20,21 @@ the unicode code point for LIGHTNING.<sup>[2](#reference-2)</sup>
 
 All messages are of form:
 
-1. `4-byte` big-endian data length.
-2. `4-byte` big-endian type.
-3. Data bytes as specified by the length.
+1. `2-byte` big-endian type.
```

Comment:
@Roasbeef we still need explicit lengths:
1) we are allowed to append data to messages in future revisions, and
2) you're supposed to totally ignore odd-typed messages you don't know.